tests/integration_tests/data_steward/utils/bq_test.py

```diff
@@ -25,8 +26,11 @@ def setUp(self):
         self.dataset_id = os.environ.get('UNIONED_DATASET_ID')
         self.description = 'Unioned test dataset'
         self.label_or_tag = {'test': 'bq'}
+        self.client = bq.get_client(self.project_id)
         # Remove dataset if it already exists
-        bq.delete_dataset(self.project_id, self.dataset_id)
+        self.client.delete_dataset(self.dataset_id,
+                                   delete_contents=True,
+                                   not_found_ok=True)
```

Comment:
Why is this line repeated in `setUp` and `tearDown`?  This seems like a bad test smell.  I would opt to just place this functionality in the `tearDown` function.

Additionally, this class could just extend `BigQueryTestBase` from `tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py` and take advantage of all the set up, clean up, and possible comparison utilities for "free".

Also, got this error when running locally,
```
======================================================================
ERROR [0.704s]: test_create_dataset (data_steward.utils.bq_test.BQTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lbiggers/Projects/curation/tests/integration_tests/data_steward/utils/bq_test.py", line 33, in setUp
    not_found_ok=True)
  File "/Users/lbiggers/Projects/env37/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 1107, in delete_dataset
    raise TypeError("dataset must be a Dataset or a DatasetReference")
TypeError: dataset must be a Dataset or a DatasetReference
```
This line is supplying a string object instead of a DatasetReference or Dataset object to the client function.  This will need to be updated in `tearDown` as well.

Reply:
I will remove the cleanup that is in setup.